pkg/util/camel/camel_dependencies.go

```diff
@@ -298,6 +298,7 @@ func addDependenciesFromCatalog(project *maven.Project, catalog *RuntimeCatalog)
 				md := maven.Dependency{
 					GroupID:    dep.GroupID,
 					ArtifactID: dep.ArtifactID,
+					Classifier: dep.Classifier,
```

Comment:
If we're having the `type` parameter, I think it makes sense to include it here as well.

Reply:
done